app/Member.php

```diff
@@ -64,8 +64,9 @@ class Member extends Model
 
     public static $options_gender = array('Male', 'Female');
     public static $options_membership_type = array('Non-member', '1 year', 'Life');
-    public static $options_allowed_membership_type = array('1 year', 'Life');
+    public static $options_allowed_membership_type = array('1 year', 'Life', 'Life (Awaiting Payment)', '1 year (Awaiting Payment)');
```

Comment:
Should "Awaiting Payment" really be a membership type, or should it be a separate field (or even table for payment history, depending on how you want to structure it)?